docs/usage.md

```diff
@@ -0,0 +1,527 @@
+  -> Expected size 83761, found 83761
+  -> Expected checksum adler32:f205f068, found adler32:f205f068
+==> Downloading file 3 of 4
+  -> File: ./5500/List_indexfile.txt
+==> Verifying file List_indexfile.txt...
+  -> Expected size 1669, found 1669
+  -> Expected checksum adler32:46a907fc, found adler32:46a907fc
+==> Downloading file 4 of 4
+  -> File: ./5500/M4Lnormdatall.cc
+==> Verifying file M4Lnormdatall.cc...
+  -> Expected size 14943, found 14943
+  -> Expected checksum adler32:af301992, found adler32:af301992
+==> Success!
+```
+
+## Listing directories
+
+The CERN Open Data files are hosted on the EOSPUBLIC data storage
+service. In order to get a list of files belonging to a certain
+EOSPUBLIC directory, please use the **list-directory** command:
+
+``` console
+$ cernopendata-client list-directory /eos/opendata/cms/validated-runs/Commissioning10
+Commissioning10-May19ReReco_7TeV.json
+Commissioning10-May19ReReco_900GeV.json
+```
+
+The **list-directory** command uses XRootD protocol to list data files
+and hence it is available only when you install the XRootD flavour.
+Please see the `installation`{.interpreted-text role="ref"}
+documentation for more details.
+
+**Iterate recursively**
+
+Note that you can use `--recursive` command-line option if you would
+like to iterate also through all the subdirectories of the given
+directory:
+
+``` console
+$ cernopendata-client list-directory /eos/opendata/cms/validated-runs --recursive
+Commissioning10-May19ReReco_7TeV.json
+Commissioning10-May19ReReco_900GeV.json
+Cert_190456-208686_8TeV_22Jan2013ReReco_Collisions12_JSON.txt
+Cert_160404-180252_7TeV_ReRecoNov08_Collisions11_JSON.txt
+Cert_136033-149442_7TeV_Apr21ReReco_Collisions10_JSON_v2.txt
+```
+
+**Iterate recursively with timeout**
+
+If you would like to list a directory that contains a large amount of
+files, you can specify `--timeout` option in order to exit after a
+certain amount of time. The default timeout is 60 seconds.
+
+``` console
+$ cernopendata-client list-directory /eos/opendata/cms/Run2010B/BTau/AOD --recursive --timeout 30
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0000_file_index.json
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0000_file_index.txt
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0001_file_index.json
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0001_file_index.txt
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0002_file_index.json
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0002_file_index.txt
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0003_file_index.json
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0003_file_index.txt
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0004_file_index.json
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0004_file_index.txt
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0005_file_index.json
+CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0005_file_index.txt
+..
+```
+
+## More information
+
+For more information about all the available `cernopendata-client`
+commands and options, please see `cliapi`{.interpreted-text role="ref"}.
```

Comment:
It seems like `interpreted-text role="ref"` is not being interpreted properly, here and elsewhere.

Before
![image](https://github.com/user-attachments/assets/b977abbf-9aaa-463a-9aea-83a3c2421b85)

After
![image](https://github.com/user-attachments/assets/ac28c03d-076c-4dcb-b9db-7479e8f6746b)